docs/Azure_IoT_Integration_Guide/Azure_IoT_Hub_integration/Device_operations/Overview.md

```diff
@@ -2,4 +2,4 @@
 
 ## How synchronization works
 
-Coiote DM provides zero-touch provisioning for synchronized devices from Azure IoT Hub. This means that device entities are automatically created within Coiote DM upon synchronization with the Azure IoT Hub and this is repeated periodically for any new devices that appear. Therefore, after one successful synchronization, you can be sure that any devices that have been added to Azure IoT Hub at a later time will also be migrated to Coiote DM.             
+Coiote DM provides zero-touch provisioning for devices synchronized with the Azure IoT Hub. This means any devices imported using a CSV file are automatically created within Coiote DM and the Azure IoT Hub and from this time on are synchronized periodically. Therefore, after any successful synchronization, you can be sure that any devices that have been imported via Coiote DM will be presented in their actual state within the Azure IoT Hub.             
```

Comment:
The same question as the one I've posted about Azure Central: do we need **the** before Azure IoT Hub in this case?

Reply:
we don't, agree